### PR TITLE
Feature/lend variable

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds Lever Up Button
- Adds FaqHeader
- Adds LendVariable directory to Lend Feature shell
- Updates borrow fixed and lend fixed to work with new TradeActionSummary component
- Updates ToggleSwitch styling
- Adds Faq to LendVariable and LendFixed pages
- Updates Provide Liquidity Faq

### CODE REVIEW

File: packages/common/mui/src/lib/faq-header/faq-header.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <ExternalLink
+              href={href}
+              textDecoration
+              accent
+              style={{ display: 'flex', alignItems: 'center' }}
+            >
+              {text}
+              <ExternalLinkIcon
+                sx={{
+                  fontSize: '12px',
```
Comment (jeffywu) [inline packages/common/mui/src/lib/faq-header/faq-header.tsx:40]: Do we need to set font size here?

File: packages/common/mui/src/lib/faq/faq.tsx [hunk 2]
```diff
@@ -113,19 +121,23 @@ export function Faq({ question, answer, slug = '' }: FaqProps) {
...
+          {questionDescription && !expanded && (
+            <Body sx={{ marginTop: theme.spacing(2) }}>
+              {questionDescription}
+            </Body>
+          )}
         </Box>
       </AccordionSummary>
       <AccordionDetails
         sx={{
           padding: '.5rem 1rem 1rem',
```
Comment (jeffywu) [inline packages/common/mui/src/lib/faq/faq.tsx:136]: Can we remove the `rem` stuff here

File: packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    </Container>
+  );
+};
+
+const Container = styled(Box)(
+  ({ theme }) => `
+    background: linear-gradient(180deg, rgba(43, 202, 208, .4) 0%, rgba(139, 193, 229, .4) 100%);
+    padding: ${theme.spacing(1)};
+    z-index: 1;
+    border-radius: 19px;
```
Comment (jeffywu) [inline packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx:44]: We can't use a theme border radius here?

File: packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  const [isChecked, setIsChecked] = useState(leveredUp);
+
+  const handleToggle = () => {
+    handleLeverUpToggle();
+    setIsChecked(!isChecked);
+  };
+
+  return (
+    <Container onClick={() => handleToggle()}>
+      <LightningIcon viewBox="0 0 12 16" sx={{ height: '16px' }} />
```
Comment (jeffywu) [inline packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx:26]: theme.spacing(2) on the height

File: packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      <H4 sx={{ textWrap: 'nowrap' }}>
+        <FormattedMessage defaultMessage={'Lever Up'} />
+      </H4>
+      <ToggleSwitch
+        isChecked={isChecked}
+        onToggle={() => handleToggle()}
+        sx={{ marginRight: '-10px' }}
```
Comment (jeffywu) [inline packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx:33]: theme.spacing(-1)

File: packages/features/lend/src/lend-fixed/lend-fixed.tsx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+                  // updateLendState({ selectedMarketKey });
+                }}
+                selectedMarketKey={selectedMarketKey || ''}
+                lockSelection={!!confirm}
+                chartToolTipData={chartToolTipData}
+              />
+              {selectedDepositToken && (
+                <TradeActionView
+                  selectedMarketKey={selectedMarketKey}
+                  tradeAction={NOTIONAL_CATEGORIES.BORROW}
```
Comment (jeffywu) [inline packages/features/lend/src/lend-fixed/lend-fixed.tsx:64]: This says BORROW

File: packages/features/lend/src/lend-fixed/lend-fixed.tsx [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+                      }}
+                    />
+                  }
+                />
+              )}
+              {selectedDepositToken && (
+                <CalculatedRatesTable
+                  selectedMarketKey={selectedMarketKey}
+                  selectedToken={selectedDepositToken}
+                  tradeAction={NOTIONAL_CATEGORIES.BORROW}
```
Comment (jeffywu) [inline packages/features/lend/src/lend-fixed/lend-fixed.tsx:94]: This also says BORROW, you can remove this component altogether

File: packages/shared/trade/src/trade-action-summary/calculated-rates-table.tsx
Comment (jeffywu) [inline packages/shared/trade/src/trade-action-summary/calculated-rates-table.tsx:1]: This component can be removed.

File: packages/shared/trade/src/deposit-input/deposit-input.tsx [hunk 8]
```diff
@@ -48,7 +48,7 @@ export const DepositInput = React.forwardRef<
   }, [updateState, inputAmount?.hashKey]);
 
   // TODO: availableDepositTokens stays undefined sometimes and causes a infinite spinner. We need to figure out why
-  if (!availableDepositTokens || !selectedDepositToken) return <PageLoading />;
+  // if (!availableDepositTokens || !selectedDepositToken) return <PageLoading />;
```
Comment (jeffywu) [inline packages/shared/trade/src/deposit-input/deposit-input.tsx:51]: Is this still a problem? Shouldn't be commented out like this?

File: packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx (hunk 3 above)
Comment (matthew-garrett) [reply packages/common/mui/src/lib/lever-up-toggle/lever-up-toggle.tsx:44]: This is based off of the mock. borderRadiusLarge from the theme is 10px which makes the button look a lot different.

**Pull request merged**